server/test/testFaceRestServer.cc

```diff
@@ -383,14 +415,34 @@ void test_getServerType(void)
 	assertStartObject(g_parser, "serverType");
 	cppcut_assert_equal(NumTestServerTypeInfo,
 	                    (size_t)g_parser->countElements());
+
+	set<size_t> remainingIndexes;
+	for (size_t i = 0; i < NumTestServerTypeInfo; i++)
+		remainingIndexes.insert(i);
+
+	auto extractIndex = [&](const int &type, const string &uuid) {
+		for (const auto idx: remainingIndexes) {
+			const ServerTypeInfo &svti = testServerTypeInfo[idx];
+			if (svti.type == type && svti.uuid == uuid) {
+				remainingIndexes.erase(idx);
+				return idx;
+			}
+		}
+		cut_fail("Not found: type: %d, uuid: %s", type, uuid.c_str());
+		return 0ul; // This line is never invoked, just to pass build.
```

Comment:
I've suggested using it for only this function.

But it is a nitpicks, I think.